src/index.html

```diff
@@ -4,14 +4,176 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input 
+          type="text" 
+          name="surname"
+          id="surname"
+          autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input 
+          type="text" 
+          name="name" 
+          id="name"
+          autocomplete="off"
+          >
+        </label>
+        
+        <label class="form-field">
+          How old are you?
+          <input 
+          type="number" 
+          name="age"
+          value="12" 
+          id="age"
+          min="1" 
+          max="100"    
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input 
+          type="date" 
+          name="birth-date" 
+          id="birth"
+          >
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement:
+          <input 
+          type="checkbox" 
+          name="agreement"
+           id="agreement"
+          >
+        </label> 
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="form-field">
+            E-mail:
+          <input 
+          type="email" 
+          name="email"
+          id="email" 
+          placeholder="email@example.com" 
+          required
+          >
+        </label>
+        <label class="form-field">
+          Password:
+          <input 
+          type="password" 
+          name="password"
+          id="password"
+          minlength="4"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+        Do you love cats?
+          <label for="yes">
+            <input 
+            type="radio" 
+            name="cats"
+            id="yes" 
+            value="yes"
+            >
+      Yes
+          </label>
```

Comment:
```suggestion
          <label>
            <input 
              type="radio" 
              name="cats"
              id="yes" 
              value="yes"
            >
            Yes
          </label>
```
Fix formatting everywhere. Also, there is no need to use "for" attribute, if input is wrapped into label